libnavigation-base/api/current.txt

```diff
@@ -1341,6 +1341,7 @@ package com.mapbox.navigation.base.trip.model.roadobject.incident {
   }
 
   public final class IncidentInfo {
+    method public java.util.List<java.lang.String>? getAffectedRoadNames();
```

Comment:
Hm, I though that github shouldn't let me merge https://github.com/mapbox/mapbox-navigation-android/pull/6008 if I don't update API descriptions. Don't we run this check on CI? 🤔